pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[project]
+name = "OpenImageIO"
+version = "2.4.0.dev1"
+description = "Reading, writing, and processing images in a wide variety of file formats, using a format-agnostic API, aimed at VFX applications."
+authors = [
+    {name = "Larry Gritz", email = "lg@larrygritz.com"},
+]
+dependencies = [
+    "setuptools>=68.2.2",
```

Comment:
Setup tools should not be needed if we use scikit-build-core. And anyway it would go into the build dependencies, not the runtime dependencies.

Reply:
Removed setuptools